src/components/annotations/convert_coords.js

```diff
@@ -26,36 +25,35 @@ var toLogRange = require('../../lib/to_log_range');
  *     same relayout call should override this conversion.
  */
 module.exports = function convertCoords(gd, ax, newType, doExtra) {
-    ax = ax || {};
+  ax = ax || {};
 
-    var toLog = (newType === 'log') && (ax.type === 'linear'),
-        fromLog = (newType === 'linear') && (ax.type === 'log');
+  var toLog = newType === 'log' && ax.type === 'linear',
+    fromLog = newType === 'linear' && ax.type === 'log';
```

Comment:
is it possible to disable this kind of change?

Using unnecessary parenthesis to improve readability is a good thing.